.github/workflows/ci.yaml

```diff
@@ -767,6 +767,27 @@ jobs:
           set -euxo pipefail
           kubectl --namespace coder rollout restart deployment/coder
           kubectl --namespace coder rollout status deployment/coder
+  
+  deploy-wsproxies:
+    runs-on: ubuntu-latest
+    needs: build
+    if: !github.event.pull_request.head.repo.fork
```

Comment:
Remember to add main check

Reply:
We can add, but I believe this is a duplication as the `build` job already has the `main` check.